packages/utils/src/bulk-insert-patients.ts

```diff
@@ -95,12 +95,12 @@ async function loadData(
   outputFileName: string,
   dryRun: boolean
 ) {
-  console.log(
-    `Loaded ${results.length} patients from the CSV file to be inserted at org/cx ${orgName}`
-  );
+  const msg = `Loaded ${results.length} patients from the CSV file to be inserted at org/cx ${orgName}`;
+  console.log(msg);
   if (dryRun) {
     console.log("Dry run, not inserting patients.");
     console.log(`List of patients: ${JSON.stringify(results, null, 2)}`);
+    console.log(msg);
```

Comment:
this gets printed twice in dryrun?

Reply:
Good catch. But it's not so odd, the issue that this change addresses is the fact that, w/o this new log, we were not seeing the first one when dry-run if we had a lot of patients. So this just logs it at the end as well, NABD.